app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -7,36 +7,25 @@ import com.example.musicapp.dto.Instrument
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import android.util.Log
 
 class InstrumentService {
-
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
         var _instruments = MutableLiveData<ArrayList<Instrument>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
         call?.enqueue(object: Callback<ArrayList<Instrument>>{
 
-            override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {
-                val j = 1 + 1
-                val i = 1 + 1
-
-            }
-
+            override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {/*...*/}
```

Comment:
Cleaned up OnFailure method to make it uncluttered from the rest of the code in InstrumentService

Reply:
This is currently an empty function.  Add a logging statement here, so that you'll at least have some way of looking up what went wrong.  Without that, if something goes wrong, this empty method will be called, won't do anything, and the user will wonder what happened.